### ISSUE TITLE
SSR - calculateInitialState

### ISSUE THREAD

**Issue by kure-** [AlecAivazis/redux-responsive#111]
Hey,
first of all, really appreaciate your work so far and thanks for your work :)

I'd have a proposal to add a prop to initialState when `calculateInitialState` is set to false while creating store. E.g. `initialized: false`. Each recalc. action of redux-responsive would set `initialized` to `true`.

It would be a great way to deal with the time e.g. on mobile devices when initial state responds to desktop. In real it renders some components with props composed to desktop and then they swap to mobile, so they look a bit different or even disappear.

Basically it would help people using SSR to not creating custom stores and dispatch actions right after manual dispatching `calculateResponsiveState` but directly just using selectors to store for that prop :) In fact you just dont want users to see that state of app and you need to display some kind of loader or skeleton :)

Thanks
